### PR TITLE
Awakeable IDs following the new ID scheme

### PR DESCRIPTION
**Pull request by AhmedSoliman**
Awakeable IDs following the new ID scheme

### CODE REVIEW

File: sdk-core/src/main/java/dev/restate/sdk/core/Entries.java [hunk 1]
```diff
@@ -19,6 +19,7 @@
 import java.util.function.Function;
 
 final class Entries {
+  public static final String AWAKEABLE_IDENTIFIER_PREFIX = "prom_1";
```
Comment (slinkydeveloper) [inline sdk-core/src/main/java/dev/restate/sdk/core/Entries.java:22]: No need for `public`

**Review by slinkydeveloper — APPROVED** [review]
Just a small nit. Once it's green, you can merge.

**Review by tillrohrmann — COMMENTED** [review]
Should we bump the protocol version to mark this change as an non-backwards compatible change?

**Comment by AhmedSoliman** [comment]
For posterity, tests pass locally but we have a full mesh of dependency such that there is no way e2e tests will pass without merging the sdk first.

**Pull request merged**